### PR TITLE
Upgrading ember

### PR DESCRIPTION
**Pull request by allthesignals**

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@ class Application < Rails::Application
     # to allow any host to connect to it. In production, set it to a comma-separated list of
     # domains where the client application is known to be hosted.
     # # => KNOWN_HOSTS=clientside.knowplace.com,another.pla.ce
-    KNOWN_HOSTS = ENV.fetch('KNOWN_HOSTS') { 'http://lvh.me:5000,http://localhost:4200,http://10.10.2.119:4200' }
+    KNOWN_HOSTS = ENV.fetch('KNOWN_HOSTS') { 'http://lvh.me:5000,http://localhost:4200,http://10.10.2.119:4200,http://0.0.0.0:4200' }
```
Comment (beechnut) [inline config/application.rb]: This line should just be `'http://lvh.me:5000,http://localhost:4200'` at most, with the rest being set in `.env`. Please remove this change, and remove `.env` from source control.

File: searchapp/app/adapters/application.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import DS from 'ember-data';
 
 export default DS.JSONAPIAdapter.extend({
   host: '//api.' + window.location.host,
+  host: 'http://api.lvh.me:5000',
```
Comment (beechnut) [inline searchapp/app/adapters/application.js]: Double-check this -- I don't think lvh.me should be hardcoded.

File: searchapp/app/routes/developments/search.js [hunk 3]
```diff
@@ -27,17 +28,18 @@ export default Ember.Route.extend({
     queryObject.page["number"] = params["number"]
     queryObject.page["size"] = params["size"]
 
-
-
-    return this.store.find("development", queryObject);
+    return this.store.query("development", queryObject).then(function(model) {
+      console.log(App);
+      model.meta = App.storeMeta['development'];
```
Comment (beechnut) [inline searchapp/app/routes/developments/search.js]: I don't usually care about Hound's JS notices, but if it's correct about this dot notation thing, I agree.

File: searchapp/app/adapters/application.js (hunk 2 above)
Comment (allthesignals) [inline searchapp/app/adapters/application.js]: Thanks, switching all this back. Developing with ember-cli in the /searchapp route messes with the current routing configuration, so I have to swap some things around with the API.

**Pull request closed without merging**